Tasks/Common/MSBuildHelpers/ArgumentFunctions.ps1

```diff
@@ -10,10 +10,13 @@ function Format-MSBuildArguments {
     Trace-VstsEnteringInvocation $MyInvocation
     try {
         if ($Platform) {
+            Test-MSBuildParam $Platform 'Platform'
             $MSBuildArguments = "$MSBuildArguments /p:platform=`"$Platform`""
         }
 
         if ($Configuration) {
+            Test-MSBuildParam $Configuration 'Configuration'
+
```

Comment:
nitpick, you have an extra empty line here you didn't have on the if-block above.